src/hooks/tradingView/useChartLines.ts

```diff
@@ -229,7 +222,25 @@ export const useChartLines = ({
         }
       }
     );
-  };
+  }, [setLineColors, stringGetter, currentMarketOrders, tvWidget]);
+
+  const drawChartLines = useCallback(
+    (widget: TvWidget) => {
+      widget.onChartReady(() => {
+        widget.headerReady().then(() => {
+          widget.chart().dataReady(() => {
+            if (showOrderLines) {
+              drawOrderLines();
+              drawPositionLines();
+            } else {
+              deleteChartLines();
+            }
```

Comment:
wonder if these are only run once when the chart is first "ready"? will let you do another pass since the order lines aren't updating t_t 

also how are position lines updated? looking at prod, the position line stays after i close the position (is that expected?)

Reply:
oof yeah this is my bad, these are intended to only run once (when chart is ready) - forgot to add back code that updates order lines when you're not switching market